examples/00-fluent/radiation_headlamp.py

```diff
@@ -0,0 +1,519 @@
+
+###############################################################################
+# Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~
+# Set the boundary conditions.
+
+# --- Set up bezel-enclosure BC ---
+# Material: plastic
+# BC type: opaque
+# Internal emissivity: 1
+# Diffuse fraction: 1
+
+bezel_enc_bc = solver.setup.boundary_conditions.wall["bezel-enclosure"]
+bezel_enc_bc.material = "plastic"
+bezel_enc_bc.radiation_bc = "Opaque"
+bezel_enc_bc.in_emiss = 1
+bezel_enc_bc.band_diffuse_frac = {"s-": 1}
+
+# Get list of wall zones
+bc_state = solver.setup.boundary_conditions.get_state()
+
+# Copy bezel-enclosure BC to all other BCs
+solver.setup.boundary_conditions.copy(
+    from_="bezel-enclosure",
+    to=bc_state["wall"].keys(),
+)
+
+# --- Set up enclosure-lens BC ---
+# Material: glass
+# BC type: semi-transparent
+# Diffuse fraction: 0
+
+enc_lens_bc = solver.setup.boundary_conditions.wall["enclosure-lens"]
+enc_lens_bc.material = "glass"
+enc_lens_bc.radiation_bc = "Semi Transparent"
+enc_lens_bc.band_diffuse_frac = {"s-": 0}
+
+# Copy enclosure-lens BC to other lens boundary
+solver.setup.boundary_conditions.copy(
+    from_="enclosure-lens",
+    to=["enclosure-lens-shadow"],
+)
+
+# --- Set up enclosure-rim-bezel BC ---
+# Material: plastic
+# BC type: opaque
+# Internal emissivity: 0.16
+# Diffuse fraction: 0.1
+
+enc_rim_bezel_bc = solver.setup.boundary_conditions.wall["enclosure-rim-bezel"]
+enc_rim_bezel_bc.material = "plastic"
+enc_rim_bezel_bc.radiation_bc = "Opaque"
+enc_rim_bezel_bc.in_emiss = 0.16
+enc_rim_bezel_bc.band_diffuse_frac = {"s-": 0.1}
+
+# Copy enclosure-rim-bezel BC to other rim bezel boundaries
+solver.setup.boundary_conditions.copy(
+    from_="enclosure-rim-bezel",
+    to=[
+        "enclosure-rim-bezel-shadow",
+        "holder-rim-bezel",
+        "holder-rim-bezel-shadow",
+        "housing-rim-bezel",
+        "housing-rim-bezel-shadow",
+    ],
+)
+
+# --- Set up enclosure:1 (domain boundaries) BC ---
+# BC type: temperature
+# Temperature: 298.15 [K]
+
+enc1_bc = solver.setup.boundary_conditions.wall["enclosure:1"]
+enc1_bc.thermal_bc = "Temperature"
+enc1_bc.t = 298.15
+
+# --- Set up radiation input BC ---
+# BC type: temperature
+# Temperature: 298.15 [K]
+# Boundary source: yes
+# Direct irradiation: 1200 [W/m^2]
+# Radiation direction: (-0.848, 0, -0.53)
+
+rad_inp_bc = solver.setup.boundary_conditions.wall["rad-input"]
+rad_inp_bc.thermal_bc = "Temperature"
+rad_inp_bc.t = 298.15
+rad_inp_bc.mc_bsource_p = True
+rad_inp_bc.band_q_irrad = {
+    "s-": {
+        "option": "value",
+        "value": 1200,
+    }
+}
+rad_inp_bc.radiation_direction = [-0.848, 0, -0.53]
+
+###############################################################################
+# Set convergence criteria
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Enable residual plots and set the convergence criteria to 'none'.
+
+solver.tui.solve.monitors.residual.plot("yes")
+solver.tui.solve.monitors.residual.criterion_type("3")
```

Comment:
Do we have settings API commands to replace these?

Reply:
I think something like this should work?
```
session.solution.monitor.residual.options.criterion_type = 3
session.solution.monitor.residual.plot() 
```

Reply:
It doesn't seem to work on my setup, any luck with yours?
![image](https://github.com/ansys/pyfluent/assets/144711134/d871958d-77ab-40e7-84ca-208baceafa6e)
![image](https://github.com/ansys/pyfluent/assets/144711134/6aa5b4d3-079d-44aa-bbdc-a16713f919e6)

Reply:
Nope, tried here and those that I identified above aren't direct equivalents. Seems like we do not have equivalents for those, thanks @ansjsia